test/testdrive/protobuf-import.td

```diff
@@ -165,26 +165,18 @@ $ kafka-create-topic topic=import-csr-wrong-proto partitions=1
 $ set-regex match="expected schema id: \d*, found \d*" replacement="expected schema id: X, found X"
 
 # Setup the correct first message, priming all the workers with the correct schema id
-$ kafka-ingest repeat=9 topic=import-csr-wrong-id format=protobuf descriptor-file=import.pb message=.Importer confluent-wire-format=true
+$ kafka-ingest repeat=100 topic=import-csr-wrong-id format=protobuf descriptor-file=import.pb message=.Importer confluent-wire-format=true
```

Comment:
why did this need to change to 100?

Reply:
because we test with a large number of workers in nightly (32), and this makes sure they all fill up

Reply:
Would it be worth adding an issue to add like {worker_count} and {worker_count_doubled} to the testdrive env?

Reply:
yeah plausibly that would be worth having.

Reply:
it would also make it easier to determine which tests care about how many workers there are.